operator/v1/types_console.go

```diff
@@ -143,8 +143,48 @@ type Capability struct {
 	Visibility CapabilityVisibility `json:"visibility"`
 }
 
+// ThemeType defines the type of the theme for the console UI.
+type ThemeType string
+
+// ThemeType values
+const (
+	// DarkTheme represents the dark theme for the console UI.
+	DarkTheme ThemeType = "Dark"
+	// LightTheme represents the light theme for the console UI.
+	LightTheme ThemeType = "Light"
+)
+
+// CustomLogoThemeFile defines a configuration based on theme types for the console UI logo.
+type CustomLogoThemeFile struct {
+	// type specifies the theme type for the console UI. It determines whether the console should use the dark or light theme.
+	// This field is an enum with valid values "Dark" and "Light".
+	// +kubebuilder:validation:Enum:="Dark";"Light"
+	// +required
+	Type ThemeType `json:"type"`
+	// path is the reference to a specific file within a ConfigMap in the openshift-config namespace.
```

Comment:
I'm confused by this. Is there a specific config map name, or is it any config map in the openshift-console namespace? This doesn't seem new, but it's not clear to me from the API doc how this works.